.github/workflows/pr-check.yaml

```diff
@@ -13,6 +13,8 @@ jobs:
         with:
           distribution: temurin
           java-version: 17
+      - name: touch local props
+        run: touch local.properties
```

Comment:
Do we need this file when running in a Github action?

Reply:
The [build was definitely failing](https://github.com/open-telemetry/opentelemetry-android/actions/runs/8942277094/job/24564512546) without this, which is why I added it. Your point below, however, about `local.properties` being automatic might be relevant here....but I'll see if I can remove the explicit handling. 

The local properties are primarily used to get a `rum.access.token` into the resource, so that it can be configured at runtime in user code. This is a hold-over from the splunk sample app -- but I think it's a nice thing to have. We might be able to just remove it and assume users have their own ways of getting an ingest token into the app. I dunno.